packages/paste-website/src/pages/components/tooltip/index.mdx

```diff
@@ -133,10 +146,10 @@ A tooltip **must** be placed on a **focusable** element, such as an anchor or bu
   }}
 >
   {`<Box display="flex" alignItems="center">
-  <Text as="span">Tooltip should appear from the icon.</Text>
-  <Tooltip text="Welcome to Paste!">
+  <Text as="span">Social Security Number</Text>
+  <Tooltip text="Your nine-digit Social Security number is your first and continuous connection with Social Security.">
     <Anchor href="https://paste.twilio.design">
-      <InformationIcon decorative={false} title="Open Tooltip" display="block" />
+      <InformationIcon decorative={false} title="More information icon" display="block" />
```

Comment:
The title here is acting as the text content of the anchor. We should be trying to describe what the anchor is doing, not what the icon is in this case